src/status_im/profile/validations.cljs

```diff
@@ -1,13 +1,17 @@
 (ns status-im.profile.validations
   (:require [cljs.spec :as s]
             [status-im.constants :refer [console-chat-id wallet-chat-id]]
+            [status-im.chat.constants :as chat-consts]
             [clojure.string :as str]
             [status-im.utils.homoglyph :as h]))
 
 (defn correct-name? [username]
-  (let [username (some-> username (str/trim))]
-    (and (not (h/matches username console-chat-id))
-         (not (h/matches username wallet-chat-id)))))
+  (let [username      (some-> username (str/trim))
+        has-cmd-char? (not= -1 (.indexOf username chat-consts/command-char))]
+    (and (not (str/blank? username))
+         (not (h/matches username console-chat-id))
+         (not (h/matches username wallet-chat-id))
+         (not has-cmd-char?))))
```

Comment:
it could look like
```clojure
(every? false? 
  [(str/blank? username)
    ...
    has-cmd-char?])

;;or

(not (or (str/blank? username)
         ...
         has-cmd-char?))
```

that's not required, but imo it looks better without repeated `not`